server/worker/tasks.py

```diff
@@ -119,11 +118,10 @@ def run_task(task: BackgroundTask, db_session=db_session) -> bool:
 
         if isinstance(error, UserError):
             logger.info(f"TASK_USER_ERROR {log_data}")
-            return True
```

Comment:
I think we still want a `return` here since we don't want to log user errors to sentry. And we also don't want to double log user errors as both a `TASK_USER_ERROR` and `TASK_ERROR`

Reply:
No prob!